com.unity.multiplayer.mlapi/Editor/NetworkingManagerEditor.cs

```diff
@@ -57,6 +57,29 @@ public class NetworkingManagerEditor : Editor
     private readonly List<Type> transportTypes = new List<Type>();
     private string[] transportNames = new string[] { "Select transport..." };
 
+    /// <summary>
+    /// Helper method that gets either the server or non-server NetworkingManager in the scene, or null if not present. 
+    /// </summary>
+    public static NetworkingManager GetNetworkingManager(bool isServer)
+    {
+        foreach (var manager in FindObjectsOfType<NetworkingManager>())
+        {
+            if (manager.IsServer == isServer)
+            {
+                return manager;
+            }
+        }
+        return null;
+    }
+
+    /// <summary>
+    /// Gets the first NetworkingManager in the scene, of any type. 
+    /// </summary>
+    public static NetworkingManager GetAnyNetworkingManager()
+    {
+        return FindObjectOfType<NetworkingManager>();
```

Comment:
While I get this is just editor code `FindObjectOfType` is one of the worst functions ever! Also assume we have more than one manager in a scene that has different settings does that matter ?

Reply:
We had similar discussions over at the [RFC-side](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/pull/11) in case you'd like to contribute :)